lib/ukrand/Makefile.uk

```diff
@@ -0,0 +1,14 @@
+$(eval $(call addlib_s,libukrand,$(CONFIG_LIBUKRAND)))
+
+CINCLUDES-$(CONFIG_LIBUKRAND)	                += -I$(LIBUKRAND_BASE)/include
+CXXINCLUDES-$(CONFIG_LIBUKRAND)                 += -I$(LIBUKRAND_BASE)/include
+
+LIBUKRAND_SRCS-$(CONFIG_LIBUKHWRAND)            += $(LIBUKRAND_BASE)/random.c
+LIBUKRAND_SRCS-$(CONFIG_LIBUKSWRAND_CHACHA)     += $(LIBUKRAND_BASE)/chacha.c
+LIBUKRAND_SRCS-$(CONFIG_LIBUKSWRAND_DEVFS)      += $(LIBUKRAND_BASE)/dev.c
+
+# TODO: Changing -y to -$(CONFIG_LIBUKSWRAND) feels natural. How to check if it this breaks anything?
```

Comment:
@michpappas, as the use of software randomness is no longer mandatory for this library, I think `getrandom()` should require `CONFIG_LIBUKSWRAND` to be selected in the config. Is my assumption correct?